templates/terraform/pre_delete/storage_hmac_key.go.erb

```diff
@@ -0,0 +1,26 @@
+getUrl, err := replaceVars(d, config, "{{StorageBasePath}}projects/{{project}}/hmacKeys/{{access_id}}")
+if err != nil {
+	return err
+}
+
+getRes, err := sendRequest(config, "GET", project, getUrl, nil)
+if err != nil {
+	return handleNotFoundError(err, d, fmt.Sprintf("StorageHmacKey %q", d.Id()))
+}
+
+// HmacKeys need to be INACTIVE to be deleted and the API doesn't accept noop
```

Comment:
Should we document this?

Reply:
I don't think we need to. Terraform will only try to take action if there's been a change, so users should never encounter this restriction.